package.json

```diff
@@ -27,7 +27,9 @@
     "build": "node bin/build",
     "coverage": "istanbul -- cover ./node_modules/.bin/nodeunit tests/unit",
     "data": "node scripts/generate-identifier-data",
-    "pretest": "jshint src && jscs src",
+    "jscs": "jscs src",
+    "jshint": "node bin/jshint src",
```

Comment:
Sorry, if you cannot see the benefit from removing jshint and using
directly, just leave it here.

Anyway, I've lost any interest in JSHint's development in any way and I
have no interest nor time to explain, obvious to me, things.

Consider my PRs closed and I'll close them later myself if no one else does
it.
On Jun 20, 2015 8:06 PM, "Luke Page" <notifications@github.com> wrote:

> In package.json
> <https://github.com/jshint/jshint/pull/2318#discussion_r32886064>:
>
> > @@ -27,7 +27,9 @@
> >      "build": "node bin/build",
> >      "coverage": "istanbul -- cover ./node_modules/.bin/nodeunit tests/unit",
> >      "data": "node scripts/generate-identifier-data",
> > -    "pretest": "jshint src && jscs src",
> > +    "jscs": "jscs src",
> > +    "jshint": "node bin/jshint src",
>
> I've glanced through the other PR's from you and I cannot see a connection.
>
> It's at least stupid to have jshint as a
> dependency for its own code.
>
> You cannot just say something is stupid and a justification. I've provided
> an example where the current setup is *more* useful than your proposed
> setup. Why is your proposed setup more useful to people developing jshint?
>
> Again you clearly miss the point.
>
> Sorry if I have, I'm just trying to sort out all the PR's, understand and
> review them. Please either explain within the PR, link to an explanation or
> you will have to rely on someone spending the time to try and understand
> what you meant, which is unlikely if you are not helpful and no benefit is
> obvious.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jshint/jshint/pull/2318/files#r32886064>.
>